server/scripts/createAITAUser.js

```diff
@@ -6,10 +6,14 @@ module.exports = async () => {
   if (AITA) {
     return;
   }
-  await User.create({
-    id: -13,
```

Comment:
looks good. im just thinking if there are any implications/problem of making the AI as a user in our database

Reply:
I don't see any problems with the AI as a user